### PR TITLE
Set up core app

### PR DESCRIPTION
**Pull request by meshy**

### CODE REVIEW

**Comment by meshy** [comment]
Please review!

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+sudo: false
+python:
+  - "3.4"
+cache:
+  - directories:
+    - ~/.cache/pip
+env:
+  global:
+    - PIP_WHEEL_DIR=$HOME/.cache/pip/wheels
+    - PIP_FIND_LINKS=file://$HOME/.cache/pip/wheels
```
Comment (Ian-Foote) [inline .travis.yml]: Are these needed with the latest pip caching?

Comment (meshy) [inline .travis.yml]: Perhaps not. If I change the cache directory, I should be able to remove them.

Comment (meshy) [inline .travis.yml]: ooh -- it looks as though `~/.cache/pip` is the default cache dir too!

**Comment by meshy** [comment]
Updated

**Comment by Ian-Foote** [comment]
LGTM

**Pull request merged**